### PR TITLE
Some changes to Readme and Cmakelists

### PR DESCRIPTION
**Pull request by DaddyWesker**

### CODE REVIEW

**Review by CICS-Oleg — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -15,18 +15,18 @@ copy mod in mods directory:
 
 now build python library
 
-`mkdir -p build && cd build && cmake .. && make`
+`mkdir -p build_cmake && cd build && cmake .. && make`
```
[the remark this answers is not in the document]
Comment (DaddyWesker) [reply readme.md:18]: Simple, build directory is created by `gradlew build` command. Thus we need separate folder. It is useful especially to clean cmake mess if something goes wrong.

File: readme.md [hunk 2]
```diff
@@ -15,18 +15,18 @@ copy mod in mods directory:
...
 
-python library is in ./build/Malmo/src/PythonWrapper/MalmoPython.so
+python library is in ./build_cmake/Malmo/src/PythonWrapper/VereyaPython.so
```
[the remark this answers is not in the document]
Comment (DaddyWesker) [reply readme.md:20]: This is due to changes in build folder name.

[the remark this answers is not in the document]
Comment (noskill) [reply readme.md:20]: you didn't change build directory name

[the remark this answers is not in the document]
Comment (DaddyWesker) [reply readme.md:20]: I mean this `mkdir -p build_cmake && cd build_cmake && cmake .. && make`

**Pull request merged**